l10n_it_esigibilita_iva/views/account_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_tax_code_esigibilita_form" model="ir.ui.view">
+        <field name="name">account.tax.form.account.tax.esigibilita</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_form" />
```

Comment:
eliminerei lo spazio finale

Reply:
è stato fatto in automatico in https://github.com/OCA/l10n-italy/pull/1938/commits/166b41cccb8d69c29eca3b49645b95bba3cc6cf5 pertanto è ok.